src/_locales/dict.setup-page.ts

```diff
@@ -2869,27 +2960,49 @@ Examples: "*", "ctrl+$", "ctrl+alt+g"`,
   'settings.snap_auto_export_type_both': {
     en: 'both',
     ru: 'оба',
+    de: 'beide',
+    zh_CN: '以上两者',
+    zh_TW: '以上兩者',
   },
   'settings.snap_export_md_tree': {
     en: 'Full-tree structure in Markdown',
     ru: 'Полная древовидная структура в Markdown',
+    de: 'Vollständige Baumstruktur in Markdown',
+    zh_CN: 'Markdown 中的完整树结构',
+    zh_TW: 'Markdown 中的完整樹結構',
   },
   'settings.snap_export_md_tree_note': {
     en: 'Add additional indents and bullets to window and panel items.',
     ru: 'Добавить дополнительные отступы и маркеры к пунктам окон и панелей.',
+    de: 'Füge zusätzliche Einrückungen und Aufzählungszeichen zu Fenster- und Panel-Elementen hinzu.',
+    zh_CN: '为窗口和面板项目添加额外的缩进和项目符号。',
+    zh_TW: '為視窗和面板項目添加額外的縮進和項目符號。',
   },
   'settings.snap_export_path': {
     en: 'Path with file name',
     ru: 'Путь с именем файла',
+    de: 'Pfad mit Dateinamen',
+    zh_CN: '路径和文件名',
+    zh_TW: '路徑和文件名',
   },
   'settings.snap_export_path_note': {
     en: `Note: Path is relative to the downloads folder.
 %Y - year; %M - month; %D - day; %h - hour; %m - minute; %s - second`,
     ru: `Примечание: Путь указывается относительно папки загрузок.
 %Y - год; %M - месяц; %D - день; %h - час; %m - минута; %s - секунда`,
+    de: `Hinweis: Pfad ist relativ zum Download-Ordner.
+%Y - Jahr; %M - Monat; %D - Tag; %h - Stunde; %m - Minute; %s - Sekunde`,
+    zh_CN: `注意：路径是相对于下载文件夹的。
+%Y - 年; %M - 月; %D - 日; %h - 小时; %m - 分钟; %s - 秒`,
+    zh_TW: `注意：路徑是相对于下載檔案夾的。
+%Y - 年; %M - 月; %D - 日; %h - 小時; %m - 分鐘; %s - 秒`,
   },
   'settings.snap_export_path_ph': {
     en: 'Sidebery/snapshot-%Y.%M.%D-%h.%m.%s',
+    ru: 'Sidebery/snapshot-%Y.%M.%D-%h.%m.%s',
```

Comment:
If there is no translation for the target language, Sidebery will fallback to 'en', so we don't need to duplicate the same value here.